policies/aws.efs.burstcreditbalance.-.value.equals.to.zero.json

```diff
@@ -0,0 +1,28 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 3,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 0,
```

Comment:
Should the level be 0? Perhaps the value should be a bit higher so users know it's very low before it's actually reaches 0. I don't feel strongly about this, just thought I'd pose the question.

Reply:
tbh for me it is not very clear from description ( https://docs.aws.amazon.com/efs/latest/ug/monitoring-cloudwatch.html )  what the  value should be set here. IMO it should be 0.